src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -226,23 +226,32 @@ static void mlpackMain()
     Timer::Start("adaboost_training");
     m->Train(trainingData, labels, numClasses, iterations, tolerance);
     Timer::Stop("adaboost_training");
+    if(IO::HasParam("test"))
+    {
+      mat testingData = std::move(IO::GetParam<arma::mat>("test"));
```

Comment:
If you use `std::move()` here, the matrix will go out of scope and the rest of the binding will fail (I assume that the CI tests fail in part because of this).

Reply:
Ok! I will try to work around it,